upp/classes/preprocessing_config.py

```diff
@@ -91,18 +100,25 @@ class PreprocessingConfig:
     out_fname: Path = Path("pp_output.h5")
     batch_size: int = 100_000
     num_jets_estimate: int = 1_000_000
+    num_jets_estimate_num: int | None = None
+    num_jets_estimate_hist: int | None = None
```

Comment:
`num_jets_estimate_num` is a pretty confusing name. I'd suggest to use

- `num_estimate_hist`
- `num_estimate_norm`
- `num_estimate_available`

Which is also nice as we shoud avoid references to "jets" where possible.

Though having three separate variables is getting unweildy here. One is bad enough since they are kinda "magic" numbers which are not untituive, so adding more makes things more confusing I think. I can see the use for making the tests pass, but do you think we really need `num_estimate_hist`?

Reply:
- I will rename it to num_estimate_available
- I think `num_estimate_hist` is maybe the most important of the 3 as one needs sufficient statistic for the histograms. Or you mean that we can remove it and just use `num_jets_estimate` instead as it was before. The problem with it is that  `num_jets_estimate` is ambiguous but I want to keep both to have backwards compatibility